cmd/podman/network_create.go

```diff
@@ -0,0 +1,69 @@
+// +build !remoteclient
+
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/adapter"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	networkcreateCommand     cliconfig.NetworkCreateValues
+	networkcreateDescription = `create networkk for containers and pods`
+	_networkcreateCommand    = &cobra.Command{
+		Use:   "create [flags] [NETWORK]",
+		Short: "network create",
+		Long:  networkcreateDescription,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			networkcreateCommand.InputArgs = args
+			networkcreateCommand.GlobalFlags = MainGlobalOpts
+			networkcreateCommand.Remote = remoteclient
+			return networkcreateCmd(&networkcreateCommand)
+		},
+		Example: `podman create podman1`,
+	}
+)
+
+func init() {
+	networkcreateCommand.Command = _networkcreateCommand
+	networkcreateCommand.SetHelpTemplate(HelpTemplate())
+	networkcreateCommand.SetUsageTemplate(UsageTemplate())
+	flags := networkcreateCommand.Flags()
+	flags.StringVarP(&networkcreateCommand.Driver, "driver", "d", "bridge", "driver to manage the network")
+	flags.IPVar(&networkcreateCommand.Gateway, "gateway", nil, "IPv4 or IPv6 gateway for the subnet")
+	flags.BoolVar(&networkcreateCommand.Internal, "internal", false, "restrict external access from this network")
+	flags.IPNetVar(&networkcreateCommand.IPRange, "ip-range", net.IPNet{}, "allocate container IP from range")
+	// TODO not supported yet
+	//flags.StringVar(&networkcreateCommand.IPamDriver, "ipam-driver", "",  "IP Address Management Driver")
+	// TODO enable when IPv6 is working
+	//flags.BoolVar(&networkcreateCommand.IPV6, "IPv6", false, "enable IPv6 networking")
+	flags.IPNetVar(&networkcreateCommand.Network, "subnet", net.IPNet{}, "subnet in CIDR format")
+
+}
+
+func networkcreateCmd(c *cliconfig.NetworkCreateValues) error {
+	if c.Driver != "bridge" {
```

Comment:
Suggestion: To pave the way for more drivers (and to have a bit more canonical go), we could add a `network.IsSupportedDriver(driver string) error`. This will prevent us from hard-coding and scattering supported drivers and also prevent us from having to change error messages.